src/main/java/hillel/doctorRest/clinic/review/dto/ReportDtoConverter.java

```diff
@@ -0,0 +1,10 @@
+package hillel.doctorRest.clinic.review.dto;
+
+import org.mapstruct.Mapper;
+
+import java.util.List;
+
+@Mapper
+public interface ReportDtoConverter {
+    List<Object> toDtoReportReview(List<Object> reportReview);
```

Comment:
What is the intention behind that? Convert a list of objects to a list of objects... Looks like this should be removed